src/handlers/comment/handlers/index.ts

```diff
@@ -45,11 +45,8 @@ export const issueClosedCallback = async (): Promise<void> => {
   if (!issue) return;
   try {
     const comment = await handleIssueClosed();
-    if (comment) await addCommentToIssue(comment, issue.number);
-    await addCommentToIssue(
-      `If you enjoy the DevPool experience, please follow <a href="https://github.com/ubiquity">Ubiquity on GitHub</a> and star <a href="https://github.com/ubiquity/devpool-directory">this repo</a> to show your support. It helps a lot!`,
-      issue.number
-    );
+    const promotionCommnet = `\n<h6>If you enjoy the DevPool experience, please follow <a href="https://github.com/ubiquity">Ubiquity on GitHub</a> and star <a href="https://github.com/ubiquity/devpool-directory">this repo</a> to show your support. It helps a lot!</h6>`;
```

Comment:
This variable is spelled wrong. Also, I'm not sure if this was originally specified, but I think this should be configurable by every repo config.

Reply:
Which one should be configurable? the site links?

Reply:
Opened new pr.

Reply:
Thank you. Next time you should link to this issue, not the other one!